api/package.json

```diff
@@ -10,7 +10,7 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "dependencies": {
-    "@polkadot/api": "4.2.1",
+    "@polkadot/api": "7.15.2-15",
```

Comment:
@MRamanenkau 
What version do we use for the latest node version? 8.x?

Reply:
fixed